### ISSUE TITLE
Markdown disappears after I publish a post

### ISSUE THREAD

**Issue by trejder** [terrylinooo/githuber-md#88]
**Describe the bug**

After I publish a post, a "Post published" info appears and page with the editor is reloaded, then suddenly WP Githubber MD stops working, whole Markdown disappears and pure HTML is displayed in editor -- even though "Enable Markdown" switch is on.

**To reproduce**

1. Go to Wordpress, start editing any new post. See that Markdown is enabled.
2. Publish this post.
3. See that Markdown is gone and HTML replaces Markdown after page with the editor is reloaded.

**Expected behavior**

As long as "Enable Markdown" switch is on I should never, ever see any piece of HTML in Markdown editor.

**Screenshots**

One screenshot depicting problem:

![Post published](https://user-images.githubusercontent.com/2903392/59263871-37002f80-8c42-11e9-9f04-c173c24bcbf2.png)

**Server environment**

 - WordPress version: **5.2.1**
 - WP Githuber MD plugin version: **1.11.4**
 - PHP version: **7.1**

**User environment**

 - OS: **Windows 10**
 - Browser: **Chrome**

**Comment by gaui** [comment]
Yep, same here.

**Comment by meibin08** [comment]
I have this problem too

**Comment by FaheyAssociates** [comment]
### Same here

#### Except that this occurs only if I leave the Edit Post window for good (close its Browser Window/Tab) and Edit again (the Publish action do not trigger the bad behavior on its own).

- WP **5.3.2**
- Githuber MD **1.15.0**
- PHP **7.2**
- We're also using [Classic Editor](https://wordpress.org/plugins/classic-editor/) WP Plugin

(I'm under OSX+Chrome)
This is **quite annoying** - not to say close to a deal-breaker: we're losing all complex formatting (shortcodes etc). 

#### And this is also weird  since it seems the MD content is saved in a special part of the DB (I would have thought safe from any manipulation?) quote:

> Your users might be confused because that if they swith to Markdown editor and then switch back to Rich editor or Gutenberg, the Markdown text of that single post will be lost. That is because that Rich editor and **Markdown editor use differnt fields to store data**.

**Comment by Li-An** [comment]
Hello, same problem here. I went to **githuber-md** because Jetpack MD module stopped to work (HTML was not generated) . I I installed the old/abandonned JPMarkdown plugin and met the same problem as Githuber-md: after saving, Markdown is not showing, HTML is showing.

It’s kinda weird as Jetpack module works on other WP on the same hosting and same database - and so do your plugin. So, for some reason, with Githuber-md and JPMarkdown the HMTL version is saved into wp_posts.post_content_filtered. I tested with no other plugin activated and default theme. It’s very frustrating as it worked nicely some weeks ago. The only difference I can see is the table size in database.
